test/objectid.test.js

```diff
@@ -101,12 +101,12 @@ describe('ObjectID', function() {
     });
   });
 
-  context("mongodb: {dataType: 'objectid'}", function() {
+  context("mongodb: {dataType: 'objectID'}", function() {
     const Article = ds.createModel(
       'ArticleC',
       {
-        xid: {type: String, mongodb: {dataType: 'objectid'}},
-        xidArr: {type: [String], mongodb: {dataType: 'objectid'}},
+        xid: {type: String, mongodb: {dataType: 'objectID'}},
+        xidArr: {type: [String], mongodb: {dataType: 'objectID'}},
```

Comment:
Let's support all three casing variants please: `ObjectID` (LB4 style), `objectid` and `objectID`.

Reply:
+1 for case insensitive